website/docs/index.html.markdown

```diff
@@ -83,6 +83,30 @@ The following arguments are supported:
 - `insights_insert_url` - (Optional) This argument changes the Insights insert URL (default is https://insights-collector.newrelic.com/v1/accounts). If the New Relic account is in the EU, the Insights API URL must be set to https://insights-collector.eu.newrelic.com/v1. The `NEWRELIC_INSIGHTS_INSERT_URL` environment variable can also be used.
 - `cacert_file` - (Optional) A path to a PEM-encoded certificate authority used to verify the remote agent's certificate. The `NEWRELIC_API_CACERT` environment variable can also be used.
 
+## Resource endpoint authentication
+
+This provider is in the midst of migrating away from our older REST based APIs to a newer GraphQL based API that we lovingly call NerdGraph.  During this transition, the provider will be using different endpoints depending on which resource is in use.  Below is a table that reflects the current state of the resources compared to which endpoint is in use.
+
+| Resource                                       | RESTv2 | NerdGraph |
```

Comment:
What do we intend to deprecate?  App labels are going away.

Reply:
Synthetics labels are going away too in mid July.  (We might want to consider deprecating APM, synthetics, plugins, and Infra conditions as well as part of the standardization efforts around NRQL conditions.  This is going to happen over a longer period of time.)